Examples/AdvancedExamples/LoaderDDS.cs

```diff
@@ -424,33 +424,34 @@ public static void LoadFromDisk( string filename, out uint texturehandle, out Te
-                                                        TextureLoaderParameters.Border,
-                                                         SurfaceSizeInBytes,
-                                                         RawDataOfSurface );
-                                break;
-                            case TextureTarget.TextureCubeMap:
-                                GL.CompressedTexImage2D( TextureTarget.TextureCubeMapPositiveX + Slices,
-                                                         Level,
-                                                         _PixelInternalFormat,
-                                                         Width,
-                                                         Height,
-                                                         TextureLoaderParameters.Border,
-                                                         SurfaceSizeInBytes,
-                                                         RawDataOfSurface );
-                                break;
-                            case TextureTarget.Texture1D: // Untested
-                            case TextureTarget.Texture3D: // Untested
-                            default:
-                                throw new ArgumentException( "ERROR: Use DXT for 2D Images only. Cannot evaluate " + dimension );
-                            }
+                            throw new NotImplementedException("AH!");
+                            //switch ( dimension )
+                            //{
+                            //case TextureTarget.Texture2D:
+                            //    GL.CompressedTexImage2D( TextureTarget.Texture2D,
+                            //                             Level,
+                            //                             _PixelInternalFormat,
+                            //                             Width,
+                            //                             Height,
+                            //                            TextureLoaderParameters.Border,
+                            //                             SurfaceSizeInBytes,
+                            //                             RawDataOfSurface );
+                            //    break;
+                            //case TextureTarget.TextureCubeMap:
+                            //    GL.CompressedTexImage2D( TextureTarget.TextureCubeMapPositiveX + Slices,
+                            //                             Level,
+                            //                             _PixelInternalFormat,
+                            //                             Width,
+                            //                             Height,
+                            //                             TextureLoaderParameters.Border,
+                            //                             SurfaceSizeInBytes,
+                            //                             RawDataOfSurface );
```

Comment:
This was busted due to the CompressedTexImage2D taking System.IntPtr instead of byte[] for the last parameter

Reply:
Also looks like the _PixelInternalFormat switched types. I attempted a fix but I'm not sure if my graphics card just doesnt support it or something, had to comment out some exceptions where it wouldnt start if I didn't have  `GL_EXT_texture_compression` extension